src/addon/file_server/template/explorer.hbs

```diff
@@ -215,8 +224,8 @@
           </span>
           <span class="bcol">{{display_name}}</span>
           <span class="bcol">{{size}}</span>
-          <span class="bcol">{{created_at}}</span>
-          <span class="bcol">{{updated_at}}</span>
+          <span class="bcol">{{date_created_string}}</span>
+          <span class="bcol">{{date_modified_string}}</span>
```

Comment:
Here you could format the date from `DateTime<Local>`. Refer: https://docs.rs/chrono/latest/chrono/format/index.html

Reply:
I've also looked at this, but this requires you to call the .format function, which you cannot do from inside the .hbs file. That't why I created separate string properties to store the result of the .format function call